packages/grid/test/drag-and-drop.common.js

```diff
@@ -254,39 +254,34 @@ describe('drag and drop', () => {
         });
       });
 
-      it('should add drag-source- part to all dragged rows', async () => {
-        grid.selectItem(grid.items[0]);
-        grid.selectItem(grid.items[1]);
+      it('should add drag-source- part to dragged rows', async () => {
         fireDragStart();
         await nextFrame();
-        for (const row of getRows(grid.$.items)) {
-          for (const cell of getRowBodyCells(row)) {
-            expect(cell.getAttribute('part')).to.contain('drag-source-row-cell');
-          }
+        for (const cell of getRowBodyCells(getRows(grid.$.items)[0])) {
+          expect(cell.getAttribute('part')).to.contain('drag-source-row-cell');
         }
       });
 
-      it('should add drag-source- part only to dragged rows', async () => {
+      it('should remove drag-source- part from dragged rows', async () => {
         fireDragStart();
-        let cells = getRowBodyCells(getRows(grid.$.items)[0]);
         await nextFrame();
-        for (const cell of cells) {
-          expect(cell.getAttribute('part')).to.contain('drag-source-row-cell');
-        }
-        cells = getRowBodyCells(getRows(grid.$.items)[1]);
-        for (const cell of cells) {
+
+        fireDragEnd();
+        await nextFrame();
+        for (const cell of getRowBodyCells(getRows(grid.$.items)[0])) {
           expect(cell.getAttribute('part')).to.not.contain('drag-source-row-cell');
         }
       });
 
-      it('should remove drag-source- part from row when drag ends', async () => {
+      it('should add drag-source- part to all dragged rows', async () => {
+        grid.selectItem(grid.items[0]);
+        grid.selectItem(grid.items[1]);
         fireDragStart();
-        const row = getRows(grid.$.items)[0];
-        const cells = getRowBodyCells(row);
         await nextFrame();
-        fireDragEnd();
-        for (const cell of cells) {
-          expect(cell.getAttribute('part')).to.not.contain('drag-source-row-cell');
+        for (const row of getRows(grid.$.items)) {
+          for (const cell of getRowBodyCells(row)) {
+            expect(cell.getAttribute('part')).to.contain('drag-source-row-cell');
+          }
```

Comment:
It's hard to read what has changed here. Have the tests been reordered?

Reply:
Not sure what happened here. I don't remember reordering them, but I suppose I did do some pretty big changes to them. What can I do to clean the diff?

Reply:
Not sure what happened here, but it seems like I did in fact reorder the tests at some point. I have now attempted to restore the order, hopefully that results in a nicer diff